x-pack/plugin/security/qa/security-basic/src/javaRestTest/java/org/elasticsearch/xpack/security/HasApplicationPrivilegesIT.java

```diff
@@ -151,6 +151,32 @@ public void testUserWithWildcardPrivileges() throws Exception {
         }
     }
 
+    public void testNamed() throws IOException {
+        createApplicationPrivilege("app", "write", new String[] { "action:write/*" });
+        createRole("correct", "app", new String[] { "read" }, new String[] { "*" });
+        createRole("wrong", "app", new String[] { "read", randomFrom("create", "write") }, new String[] { "*" });
```

Comment:
Only difference between the roles: "single" unknown privilege or unknown privilege + plus another (known or unknown, doesn't matter).